src/helpers/index.js

```diff
@@ -113,18 +113,14 @@ export default {
 
 		return niceFraction * Math.pow(10, exponent);
 	},
-	// Request animation polyfill - https://www.paulirish.com/2011/requestanimationframe-for-smart-animating/
+	// Request animation polyfill
 	requestAnimFrame: (function() {
 		if (typeof window === 'undefined') {
 			return function(callback) {
 				callback();
 			};
 		}
 		return window.requestAnimationFrame ||
-			window.webkitRequestAnimationFrame ||
-			window.mozRequestAnimationFrame ||
-			window.oRequestAnimationFrame ||
-			window.msRequestAnimationFrame ||
 			function(callback) {
```

Comment:
I think the `setTimeout` version can be removed too, right?

Reply:
Yeah, good call